include/config/config_rom.h

```diff
@@ -19,23 +19,45 @@
 
 /**
  * Enables Rumble Pak Support.
- * Currently not recommended, as it may cause random crashes.
  */
-// #define ENABLE_RUMBLE (1 || VERSION_SH)
+#define ENABLE_RUMBLE
 
 /**
  * The maximum number of supported players/controllers. 1-4.
  * This will save performance if the player has extra unused controllers plugged in.
- * NOTE: Default is 2, maximum is 4.
- * NOTE: This needs to be at least 2 for now for gamecube controller swap to work.
+ * NOTE: Default is 1, maximum is 4.
+ * NOTE: If this is 1, the first controller with input after boot will become player 1. Otherwise, player numbers are automatically assigned via port order.
+ * NOTE: It is highly recommended to enable ENABLE_STATUS_REPOLLING_GUI if this is greater than 1.
  */
-#define MAX_NUM_PLAYERS 2
+#define MAX_NUM_PLAYERS 1
+
+/**
+ * Enables a GUI that, while opem, allows plugging/unplugging controllers and changing the controller order.
+ * Triggered by unplugging any active controller.
+ */
+#define ENABLE_STATUS_REPOLLING_GUI
+
+/**
+ * Allows the player to use the button combo defined in input.h to trigger status repolling at any time during gameplay.
+ */
+#define ENABLE_STATUS_REPOLLING_COMBO_IN_GAMEPLAY
+
+/**
+ * Highlights the currently pressed buttons on controllers in the status repolling GUI.
+ */
+#define CONTROLLERS_INPUT_DISPLAY
```

Comment:
why are you adding another 3 tiny random defines that literally no-one is going to know what they do. this is awful lol we're trying to avoid exactly this